src/js/components/Carousel/StyledCarousel.js

```diff
@@ -0,0 +1,82 @@
+import styled, { css, keyframes } from 'styled-components';
+import { Box } from '../Box';
+import { widthStyle, heightStyle } from '../../utils';
+
+const StyledCarouselContainer = styled(Box)`
+  position: relative;
+  display: block;
+  overflow: hidden;
+  height: 100%;
+  ${(props) => props.height && heightStyle(props.height, props.theme)}
+  ${(props) => props.width && widthStyle(props.width, props.theme)}
+  ${(props) =>
+    props.inTransition &&
```

Comment:
Since `containerHeight` and `containerWidth` are already undefined when not `inTransition`, do we still need to test for `props.inTransition` here?